### DISCUSSION TITLE
How give general access to Vercel preview

### DISCUSSION THREAD

**Discussion by benediktbrenk** [neuefische/questions#830] [Web] [answered]
Hey, we have just started our capstone project. When others wanted to access the Vercel preview of our project, we were asked to give access. How is it possible to give general access to the preview? Thanks :)

**Comment by MerleMak** [comment]
You are on the _Vercel Overview Page_ > Select your Capstone Project > _Project Overview Page_ > Select Settings > _Project Settings_ > Select Deployment Protection > Toggle the switch button under "_Vercel Authentication_" to "disabled" > Done

**Comment by F-Kirchhoff** [comment, marked as answer]
Here is a little guide on how to disable the deployment protection:

![disable-deploymet-protection](https://github.com/neuefische/questions/assets/59687330/a0f861dc-c4bb-454b-af65-075e2810d3bd)
